dillorc

```diff
@@ -195,6 +195,11 @@ search_url="Google http://www.google.com/search?ie=UTF-8&oe=UTF-8&q=%s"
 # HSTS directives are not saved between browser sessions.
 #http_strict_transport_security=YES
 
+# If enabled, Dillo will force all all HTTP connections to be upgraded to a
+# more secure HTTPS connection.
+# This option will prevent sites from loading if they only support HTTP.
```

Comment:
Consider merging this line into the previous paragraph, as it only occurs when the option is enabled:
> If enabled, Dillo will force all HTTP connections to be upgraded to a more secure HTTPS connection, which will prevent sites from loading if they only support HTTP.